src/effects/Reverb.cpp

```diff
@@ -383,20 +376,21 @@ bool EffectReverb::DoLoadFactoryPreset(int id)
       return false;
    }
 
-   mSettings = FactoryPresets[id].preset;
+   // TODO - ::LoadFactoryPreset should belong to the instance,
+   // but the framework is not ready for that yet
+   // 
+   // mSettings = FactoryPresets[id].preset;
```

Comment:
The left side of the assignment simply changes to `GetSettings(settings)` and you no longer need Const_cast and a Do... function.

That eliminates all const_cast from this source file, a good thing to mention in the commit comment.

Reply:
Ok - I was wrong, the framework was not missing something, rather I did not fully understand it. I did how you suggest above (actually - before reading your comment; good that I came to realize it by myself!) and I verified that presets are correctly loaded.